galley/cmd/galley/cmd/root.go

```diff
@@ -18,27 +18,42 @@ import (
 	"flag"
 	"fmt"
 	"time"
-
 	"github.com/spf13/cobra"
-	"github.com/spf13/cobra/doc"
-
 	"istio.io/istio/galley/cmd/shared"
-	"istio.io/istio/pkg/collateral"
 	"istio.io/istio/pkg/log"
+	"istio.io/istio/galley/pkg/server"
+	galleyProbe "istio.io/istio/galley/pkg/probe"
+	"istio.io/istio/galley/pkg/crd/validation"
+	istiocmd "istio.io/istio/pkg/cmd"
+	"istio.io/istio/pkg/collateral"
+	"github.com/spf13/cobra/doc"
 	"istio.io/istio/pkg/version"
+
+	"istio.io/istio/pkg/probe"
 )
 
 var (
 	flags = struct {
 		kubeConfig   string
 		resyncPeriod time.Duration
+		noValidation         bool
+		noServer             bool
+		livenessProbeOptions *probe.Options
+		readinessProbeOptions *probe.Options
```

Comment:
I would declare `livenessProbeOptions` and `readinessProbeOptions` as value types instead of pointers and move it to within the scope of `GetRootCmd` (line ~56). This would eliminate lines 104-105 and the use of global variables.

Reply:
Done